com.gitlab.coringao.cavestory-nx.json

```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "com.gitlab.coringao.cavestory-nx",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cavestory-nx",
+  "finish-args": [
+    "--device=all",
+    "--socket=pulseaudio",
+    "--socket=x11"
+  ],
+  "modules": [
+    {
+      "name": "cavestory-nx",
+      "no-make-install": true,
+      "buildsystem": "cmake-ninja",
+      "build-commands": [
+        "install -D cavestory-nx /app/bin/cavestory-nx",
+        "cp -R data /app/bin/data",
+        "install -Dm644 cavestory-nx.desktop /app/share/applications/com.gitlab.coringao.cavestory-nx.desktop",
+        "install -Dm644 data/cavestory-nx.png /app/share/icons/hicolor/512x512/apps/com.gitlab.coringao.cavestory-nx.png",
+        "install -Dm644 com.gitlab.coringao.cavestory-nx.appdata.xml /app/share/appdata/com.gitlab.coringao.cavestory-nx.appdata.xml"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://gitlab.com/coringao/cavestory-nx.git",
+          "tag": "1.3.0",
+          "commit": "30509c41e5b0537acbcee6f0114f91967a5365ec"
+        },
+        {
+          "type": "file",
+          "path": "cavestory-nx.desktop"
```

Comment:
Why not `com.gitlab.coringao.cavestory-nx.desktop`?

Reply:
I'm not saying it's bad, but then he could use this:
```
install -Dm644 ${FLATPAK_ID}.desktop -t /app/share/applications/
```
instead of this:
```
install -Dm644 cavestory-nx.desktop /app/share/applications/com.gitlab.coringao.cavestory-nx.desktop
```
The shorter, the better.